frontend_lib/src/component/Timeline/CommentFilePreview.jsx

```diff
@@ -2,32 +2,118 @@ import React from 'react'
 import { translate } from 'react-i18next'
 import PropTypes from 'prop-types'
 import classnames from 'classnames'
+import Lightbox from 'react-image-lightbox'
+import AttachedFile from '../AttachedFile/AttachedFile.jsx'
 import {
   buildFilePreviewUrl,
   removeExtensionOfFilename,
-  getFileDownloadUrl
+  getFileDownloadUrl, handleFetchResult
 } from '../../helper.js'
-import AttachedFile from '../AttachedFile/AttachedFile.jsx'
+import { getFileRevisionPreviewInfo } from '../../action.async'
```

Comment:
I think the usage is more to include the extension when importing a local file:
```suggestion
import { getFileRevisionPreviewInfo } from '../../action.async.js'
```